package.json

```diff
@@ -50,8 +50,12 @@
     "test": "istanbul cover -x '**/*.test.js' node_modules/mocha/bin/_mocha '!(node_modules)/**/*.test.js' -- -R spec --recursive",
     "lint": "eslint .",
     "docs": "node scripts/generate-readme.js",
-    "simple-integration-test": "mocha tests/integration/simple-integration-test",
-    "complex-integration-test": "mocha tests/integration/all"
+    "simple-integration-test": "jest --maxWorkers=5 simple-suite",
```

Comment:
absolutely, do you mind if I do it a next PR? I was worried that this one would get too big and hard to review therefor.